src/Microsoft.TemplateEngine.VisualStudioWizard/Microsoft.TemplateEngine.VisualStudioWizard.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <TargetFramework>$(NETFullTargetFramework)</TargetFramework>
+        <EnableAnalyzers>true</EnableAnalyzers>
```

Comment:
This project is not used anymore. Probably we should remove it from repo

Reply:
I created a new task for it here: https://github.com/dotnet/templating/issues/2796